pandas/tests/plotting/test_datetimelike.py

```diff
@@ -5,9 +5,13 @@
 
 import pytest
 from pandas.compat import lrange, zip
+try:
```

Comment:
why are you adding this? we already have machinery to import correctly

Reply:
Purely for debugging reasons, because I wanted to display `matplotlib.__version__` if things fail